libpod/info.go

```diff
@@ -0,0 +1,120 @@
+package libpod
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"runtime"
+
+	"github.com/docker/docker/pkg/system"
+)
+
+type InfoData struct {
+	Type  string
+	Data  map[string]interface{}
```

Comment:
I believe Ryan made this map[string]interface{} because the data being stored are of different types. So no, unless we want to go through and convert everything to strings.